infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/workflows/RemoveNodeWorkflow.java

```diff
@@ -62,6 +64,7 @@ public String getName() {
 
         @Override
         public void impl(@Nonnull CorfuRuntime runtime) throws Exception {
+            runtime.invalidateLayout();
```

Comment:
The orchestrator will pass down a connected client (i.e. it will have a layout ready) when it creates a workflow, why is this needed?

Reply:
Fixed